app/templates/components/request/configs.hbs

```diff
@@ -0,0 +1,101 @@
+<div class="request-configs {{this.status}}">
+  {{#if this.showNewConfigView}}
+    {{#if this.canTriggerBuild}}
+      <Request::ConfigsButtons
+        @onCancel={{action this.onCancel}}
+        @onCustomize={{action this.onCustomize}}
+        @onPreview={{action this.onPreview}}
+        @onTrigger={{action this.onTrigger}}
+        @closed={{this.closed}}
+        @submitting={{this.submitting}}
+      />
+    {{/if}}
+
+    {{#if (or this.closed this.open)}}
```

Comment:
This looks a bit confusing to me 😕 From how it reads, the condition should always be `true`. Just wanted to bring your attention, to make sure, the logic is correct here.

Reply:
ah, i see what you mean, haha :)

hmmm, the component can be in the states `closed`, `open`, `customizing`, and `previewing`. i've gone back and forth between various combinations of namings, but couldn't come up with a better version than this.

we could of course move this to the component and give it a property `super-hard-to-name`, but i'm not sure that would be an improvement ...

i'd probably keep it as is?

Reply:
Yea, it works perfectly, I just wanted to make sure it's not a mistake :)